src/main/kotlin/no/nav/familie/ef/mottak/integration/SøknadClient.kt

```diff
@@ -37,29 +50,31 @@ class SøknadClient(@Qualifier("restTemplateAzure") operations: RestOperations,
         } catch (e: RestClientException) {
             throw IllegalStateException("Innsending til sak feilet.", e)
         }
-
     }
 
-    fun sendTilSak(skjemasak: Skjemasak): HttpStatus? {
+    fun send(skjemasak: Skjemasak): HttpStatus? {
         log.info("Sender søknad til {}", sendInnSkjemasakUri)
 
         try {
-            return postForEntity(sendInnSkjemasakUri, skjemasak)
+            throw RuntimeException("Denne endepunkten finnes ikke ennå.")
+            //return postForEntity(sendInnSkjemasakUri, skjemasak)
         } catch (e: RestClientResponseException) {
             log.warn("Innsending til sak feilet. Responskode: {}, body: {}",
                      e.rawStatusCode,
                      e.responseBodyAsString)
-            throw IllegalStateException("Innsending til sak feilet. Status: ${e.rawStatusCode}, body: ${e.responseBodyAsString}",
-                                        e)
+            throw IllegalStateException(
+                    "Innsending til sak feilet. Status: ${e.rawStatusCode}, body: ${e.responseBodyAsString}", e)
         } catch (e: RestClientException) {
             throw IllegalStateException("Innsending til sak feilet.", e)
         }
-
     }
 
+    private fun urlBuilder(sakConfig: SakConfig) = DefaultUriBuilderFactory().uriString(sakConfig.url)
+
     companion object {
-        const val PATH_SAK = "sak/sendInn"
-        const val PATH_SKJEMASAK = "skjemasak/sendInn"
+        const val PATH_OVERGANGSSTØNAD = "external/sak/overgangsstonad"
+        const val PATH_BARNETILSYN = "external/sak/barnetilsyn"
+        const val PATH_ARBEIDSSØKER = "external/sak/arbeidssoker"
```

Comment:
Vi sender ikke inn noe til sak ennå og har en tanke om att Controllert som mottar disse skal kun være for innsending, med external i pathen, for å ikke blande API'iet til GUI og det som sendes inn

Reply:
Godt poeng. Ser ut som vi sender til sak i `dev`, men det kan vi kanskje bare skru av?

Reply:
Yep, skall fikse den nå :)